.github/workflows/markdown.links.config.json

```diff
@@ -5,6 +5,9 @@
     },
     {
       "pattern": ".*https://dev.azure.com/Lightning-AI/lightning/_apis/build/status/.*"
+    },
+    {
+      "pattern": "https://pytorch-lightning.readthedocs.io/en/latest/fabric/fabric.html"
```

Comment:
I think that in general, we shall rather to `stable` versions as the `latest` does not mean that will be really released (it can be reverted any time)